docs/monitoring/tail-log-file.md

```diff
@@ -0,0 +1,24 @@
+---
+title: tail -f /var/log/my-application.log
+description: "Aggregate log files from different pods"
```

Comment:
Why do you think this guides needs to be in g/g?

Reply:
Same as above.

Reply:
I don't see much value in this document. And I would move it to `/dev/null`.